### PR TITLE
Edit UG

### PR DESCRIPTION
**Pull request by VedJoshi**
fixes
- #260
- #254
- #252

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Bridal Boss is a **desktop app for managing contacts, optimized for use via a  L
 #### Email Addresses
 
 - **Format**:
-    - Must be in the form `local-part@domain`.
+    - Must be in the form `local-part@domain.com`.
```
Comment (jowhee3011) [inline docs/UserGuide.md:110]: Email regex does not correspond to ending with .com, perhaps we can say end with xx, where x is an  alphanumeric character?

**Review by jowhee3011 — COMMENTED** [review]
generall lgtm except for email requirements

File: docs/UserGuide.md (hunk 1 above)
Comment (VedJoshi) [reply docs/UserGuide.md]: latest commit should be more clear

File: docs/UserGuide.md [hunk 2]
```diff
@@ -162,6 +277,7 @@ Certain commands (`edit`, `delete`, `deletew`, `view`, `vieww`, `assign`) suppor
 
 - **Usage**:
     - Uses the position number from the displayed list.
+    - Only integers are accepted.
```
Comment (yikjunxian) [inline docs/UserGuide.md:280]: only positive, non-zero integers are accepted

**Review by yikjunxian — COMMENTED** [review]
Minor tweak for valid index.

Also perhaps sections like validation rules, index vs name based commands, cross-validation can be put below general command details. Cuz otherwise users have to read through quite a lot cluster before they can even start to read about the specific command.

Also also might just be a personal opinion but perhaps the parameters in the common scenarios section can be omitted cuz i feel like it'll overwhelm users if they dont even know how our commands work yet. If the point is to let users have some intuition to how to use our app maybe can just include the command word only. E.g.
1. First add the client as a contact using `add` along with their relevant information
2. Next, create their wedding using `addw`
3. Lastly their wedding can be viewed using `vieww`

**Comment by VedJoshi** [comment]
> Minor tweak for valid index.
> 
> Also perhaps sections like validation rules, index vs name based commands, cross-validation can be put below general command details. Cuz otherwise users have to read through quite a lot cluster before they can even start to read about the specific command.
> 
> Also also might just be a personal opinion but perhaps the parameters in the common scenarios section can be omitted cuz i feel like it'll overwhelm users if they dont even know how our commands work yet. If the point is to let users have some intuition to how to use our app maybe can just include the command word only. E.g.
> 
> 1. First add the client as a contact using `add` along with their relevant information
> 2. Next, create their wedding using `addw`
> 3. Lastly their wedding can be viewed using `vieww`

I think giving example commands lets them see how it works faster, the user can figure out the command inputs faster that way - reordering makes sense, i'll do that.

**Pull request merged**